css/style.css

```diff
@@ -1,15 +1,455 @@
+* {
+	box-sizing: border-box;
+}
+
+html {
+	overflow-x: hidden;
+}
+
 body {
 	font-family: 'GothamPro Bold';
+	margin: 0;
+	padding: 0;
 }
 
 a {
 	text-decoration: none;
+	display: inline-block;
+}
+
+/*header start*/
+.header-content {
+	max-width: 1170px;
+	margin: 0 auto;
 }
 
-.header__container {
+.header-container {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
 	max-width: 1030px;
+	height: 115px;
+	margin: 0 auto;
+}
+
+.header-container__inner {
+	flex: 1;
+	display: flex;
+	justify-content: flex-start;
+	align-items: center;
+}
+
+.header-logo {
+	width: 55px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project_Axion/pullRequest?prid=5457136)